<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoomPlot</class>
 <widget class="QDialog" name="zoomPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zoom Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#zoomPlot{
	background-color: rgb(0, 0, 0);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>670</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Display Plot:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="hCutPos">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>630</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="plotTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>332</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#838383;&quot;&gt;Plotter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gradient">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Directional Gradient</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#gradient{
image:url(:/nSOTScanner/Pictures/inverseNabla.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#gradient{
image:url(:/nSOTScanner/Pictures/inverseNabla.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="mainPlotArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(132, 132, 132);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="hideGrad">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>45</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Hide Color Bar</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="XZPlotArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>530</y>
     <width>635</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(132, 132, 132);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="diamFrame">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>70</y>
     <width>120</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#diamFrame{background-color: rgb(0, 0, 0);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="checkFourier">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>13</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>10</y>
      <width>91</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Fourier Over ROI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkAvg">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>180</y>
      <width>13</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>180</y>
      <width>86</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Manual Average&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fourierEst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>27</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;(nm)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>115</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Diameter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourierCalc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#fourierCalc{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#fourierCalc{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>87</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Select ROI and&lt;br/&gt;press calculate. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="avgAddLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#avgAddLine{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#avgAddLine{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>27</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;(nm)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>345</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Diameter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="avgEst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>365</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>85</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#838383;&quot;&gt;Add lines to &lt;br/&gt;average over&lt;br/&gt;desired number&lt;br/&gt;of periods.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="avgCalc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#avgCalc{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#avgCalc{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="vCutPos">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>570</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>550</y>
     <width>99</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Vertical Line-Cut:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>610</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Horizontal Line-Cut:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vhSelect">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>690</y>
     <width>97</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertical</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="subtract">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>17</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Subtract Fit</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#subtract{menu-indicator:{image:none}}

#subtract{
image:url(:/nSOTScanner/Pictures/inverseRes.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#subtract{
image:url(:/nSOTScanner/Pictures/inverseRes.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sensitivity">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Compute Sensitivity</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#sensitivity{
image:url(:/nSOTScanner/Pictures/inverseSigma.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#sensitivity{
image:url(:/nSOTScanner/Pictures/inverseSigma.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="YZPlotArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>530</y>
     <width>635</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(132, 132, 132);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="showGrad">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>45</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Show Color Bar</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="plotType">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Refresh Plot</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#refresh{
image:url(:/nSOTScanner/Pictures/refresh.png);
background: black;

}

QPushButton:pressed#refresh{
image:url(:/nSOTScanner/Pictures/refresh.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;X-Axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yAxis">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Y-Axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Z-Axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="zAxis">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="xAxis">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>17</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Plot original data selection</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="savePlot">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>17</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Compute Sensitivity</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#savePlot{
image:url(:/nSOTScanner/Pictures/save.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#savePlot{
image:url(:/nSOTScanner/Pictures/save.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>YZPlotArea</zorder>
  <zorder>label</zorder>
  <zorder>hCutPos</zorder>
  <zorder>plotTitle</zorder>
  <zorder>gradient</zorder>
  <zorder>mainPlotArea</zorder>
  <zorder>hideGrad</zorder>
  <zorder>XZPlotArea</zorder>
  <zorder>diamFrame</zorder>
  <zorder>vCutPos</zorder>
  <zorder>label_27</zorder>
  <zorder>label_29</zorder>
  <zorder>vhSelect</zorder>
  <zorder>subtract</zorder>
  <zorder>sensitivity</zorder>
  <zorder>showGrad</zorder>
  <zorder>plotType</zorder>
  <zorder>refresh</zorder>
  <zorder>label_2</zorder>
  <zorder>yAxis</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>zAxis</zorder>
  <zorder>xAxis</zorder>
  <zorder>back</zorder>
  <zorder>savePlot</zorder>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
